app/views/comments/deleted_commentable_comment.html.erb

```diff
@@ -0,0 +1,63 @@
+<div class="single-comment-header"></div>
```

Comment:
Sorry if I didn't notice this in the beginning, but the shouldn't the view render the entire comment tree (see what happens in (comments/index), not just a single comment?

Also there's a lot of metadata in the index view that we might need here as well. Maybe this should just be ap partial rendered by index in case commentable is nil, instead of rendering a separate view in the controller

Reply:
@rymes and @mstruve no worries on the delay. 

> shouldn't the view render the entire comment tree (see what happens in (comments/index), not just a single comment?

I had originally discussed this with mstruve. I was under the impression that you wanted a read only view to show the single comment without any actions on it(liking, replying, etc.  
Would you want the entire tree shown this way after the article/podcast is deleted?  To me, the comment more belongs to the user and the full comment tree more belongs to the article. I could change it to add the full comment tree but a lot of that functionality depends on the commentable(like the tree_for helper method and the _comment_proper partial).

> Also there's a lot of metadata in the index view that we might need here as well. Maybe this should just be a partial rendered by index in case commentable is nil, instead of rendering a separate view in the controller

I could add the meta data for the root_comment but a lot of the meta data is for the commentable.  I could make it a partial but a lot of the index view requires a commentable so you would need to wrap it in a conditional.

I had mentioned earlier, I think the best option might be to add a separate show action in the comments controller and redirect to it if there is no commentable. The show action would not require commentable and just display the single comment.  Let me know what you think.

Reply:
> I had originally discussed this with mstruve

@rhymes I think just getting the single comment to load properly is a great start since currently, it doesn't even do that and then we can save loading the entire tree and other meta data for another PR.

Reply:
Thanks for taking the time to explain it to me @twinsfan421, deeply appreciated and sorry if I didn't understand the full scope before. @mstruve agreed! Let's start from somewhere!

Reply:
@rhymes no worries, thanks for taking the time to review. There is a lot going on with the index action and views and so much of it is tied to the commentable. I am happy to change things up or go in a different direction if preferred.